src/subs.ts

```diff
@@ -0,0 +1,30 @@
+import { roleMention } from "discord.js";
+
+import { discordClient } from "./clients/discord.js";
+
+export async function rollSubs() {
+  const GUILD_ID = process.env.GUILD_ID!;
```

Comment:
Aren't these traditionally top level for most discord client things?

Why aren't they here?

Reply:
Because of a quick refactor. Will fix later as I'm planning on revamping config variables soon